Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md

```diff
@@ -0,0 +1,207 @@
+<a name="br1"></a> 
+
+# *Detail Design: CO2 and NO2 Sensor*
+
+## **Function of the Subsystem:**
+
+  The function of this subsystem is to relay the CO2 and NO2 levels to the Arduino via their
+
+respecitive sensors. The subsystem will use a series of sensors to transfer the percentage levels
+
+of CO and NO in the air to the Arduino. The sensors will constantly monitor the changes of the
+
+carbon dioxide and nitrogen dioxide levels in the greenhouse’s atmosphere and will wirelessly
+
+report these numbers and changes to the Arduino via Bluetooth.
+
+## **Constraints:**
+
+- **Carbon Dioxide Sensor – MH-Z19B**
+
+  - MH-Z19B needs between 4.5-5.5V and at least 20mA to operate eﬃciently.
+
+  - The sensor must turn on every 5 minutes to measure the amount of carbon
+
+    dioxide present in the greenhouse.
+
+  - Must alert when above or below 400-2000ppm to ensure proper plant growth.
+
+- **Arduino Nano 33 BLE**
+
+  - The communication between the sensors and PLC must be wireless. This will save
+
+    time and money on wiring, while also allowing the system to be scalable.
+
+  - The transfer of data between the sensors and the Arduino must be nearly
+
+    instantaneous for accurate and reliable monitoring of the CO and NO levels in
+
+    the atmosphere.
+
+  - BLE (Bluetooth Low Energy) is lower in cost than the classic Bluetooth as BLE is
```

Comment:
And it's not compatible with many Bluetooth devices... probably ought to check into that.